lib/globalize.js

```diff
@@ -0,0 +1,5 @@
+var path = require('path');
+var SG = require('strong-globalize');
+
+SG.SetRootDir(path.join(__dirname, '..'), { autonomousMsgLoading: 'all' });
```

Comment:
It passes the CI PR Linter. Eslint is not configured for this repo yet. Can fix this when I add eslint.